automation/submission-perf-inc.sh

```diff
@@ -184,12 +196,15 @@ findSubmissionID() {
     fi
     submissionID=$(
         curl \
+            --retry 3 \
+            --retry-max-time 20 \
             -X GET \
             --header 'Accept: application/json' \
             --header "Authorization: Bearer ${ACCESS_TOKEN}" \
             "https://firecloud-orchestration.dsde-alpha.broadinstitute.org/api/workspaces/$namespace/$name/submissions" \
+        | tee submissionIDResponse.json \
         | jq -r "[.[] | select($selectorString)] | sort_by(.submissionDate) | reverse[0] | .submissionId")
-
+    cat submissionIDResponse.json
```

Comment:
When this test has flaked/failed in the past, this HTTP request was the culprit, but the error was passed into `jq` and lost. This change should dump the failed response into the logs so we have a lead to chase when troubleshooting.